dashboard/api.py

```diff
@@ -91,20 +81,19 @@ def get_assoc_field(cls, course_status):
         """
         Method to get from the ASSOCIATED_FIELDS dict
         """
-        if course_status not in CourseStatus.all_statuses():
-            raise ImproperlyConfigured(
-                '{} not defined in Courses.api.CourseStatus'.format(course_status))
+        if course_status not in RunStatus.all_statuses():
+            raise ImproperlyConfigured('{} not defined in Courses.api.CourseStatus'.format(course_status))
         return cls.ASSOCIATED_FIELDS.get(course_status, [])
 
 
-class CourseRunUserStatus:
+class UserCourseRun:
     """
-    Representation of a course run status for a specific user
+    Representation of a course run for a specific user
     """
-    def __init__(self, status, course_run=None, enrollment_for_course=None):
-        self.status = status
+    def __init__(self, course_run, status, certificate=None):
```

Comment:
this ordering just made more sense to me. the CourseRun object is (in my opinion) the 'operative'/most important property of this class, and to me it's weird for the order of argument order to be `(string, complex_model_object, ...)`